server/test-requirements.txt

```diff
@@ -6,3 +6,4 @@ pytest>=4.6.3, < 5
 pytest-helpers-namespace>=2019.1.8
 pytest-mock>=1.10.4
 requests_mock
+selinux
```

Comment:
You don't need this in `test-requirements.txt` if you already have it in `requirements.txt`.

Reply:
oh ok... I will change that

Reply:
I see that we have resolved this by using an RPM in the development container.

We also need to then add this in the pbench-server RPM `.spec` file.

But, we should also have a python3 library module requirement in the `server/requirements.txt` file.  Can that be restored?